app/src/org/commcare/activities/SyncCapableCommCareActivity.java

```diff
@@ -45,6 +46,7 @@
     private SyncIconState syncStateForIcon;
     private SyncIconTrigger lastIconTrigger;
     private MenuItem currentSyncMenuItem;
+    private boolean showRateLimitDialog = true;
```

Comment:
This should depend on a custom property from the application settings on HQ so that we can disable it selectively for preojects like ICDS.

Reply:
Got it! Sounds good.